package.json

```diff
@@ -1,20 +1,51 @@
 {
   "name": "css-frameworks-ca",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Not a major version change as no breaking changes have been introduced.

This is either 1.1.0 or 1.0.x depending on your approach.